### ISSUE TITLE
[PATCH] Makefile colors doesn't work in Linux and FreeBSD

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ashish21590/redis#580]
```
What version of Redis you are using, in what kind of Operating System?
Latest unstable branch in Linux and FreeBSD.

What is the problem you are experiencing?
The color codes are printed as text instead of generating colors.

What steps will reproduce the problem?
Compile redis.

Please provide any additional information below.

echo doesn't always interpret escape sequences by default. echo -e works in all 
Linux and FreeBSD shells that I've tried, but seems to be a non-standard 
extension. printf should work on all POSIX systems [1], so it's probably the 
best solution.

Fix available here: https://github.com/hampus/redis/tree/fix-makefile-colors. 
It simply uses printf instead. Tested in Arch Linux and FreeBSD 8.2.


[1] http://pubs.opengroup.org/onlinepubs/009695399/utilities/printf.html
```

Original issue reported on code.google.com by `hampus.w...@gmail.com` on 9 Jun 2011 at 1:08

**Comment by GoogleCodeExporter** [comment]
```
I believe this has been merged here: 
https://github.com/antirez/redis/commit/db6cea11c4164423c318ec2a4ad1c42a273feaaf
```

Original comment by `pcnoordh...@gmail.com` on 13 Jun 2011 at 2:11
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
```
Yep. It just wasn't closed before.
```

Original comment by `hampus.w...@gmail.com` on 13 Jun 2011 at 2:33

**Comment by GoogleCodeExporter** [comment]
```
Merged indeed, also backported 2.4. Thanks both.
```

Original comment by `anti...@gmail.com` on 13 Jun 2011 at 7:24
